node-wasm/src/worker.rs

```diff
@@ -48,12 +39,77 @@ pub enum WorkerError {
     NodeError(Error),
 }
 
+#[wasm_bindgen]
 struct NodeWorker {
+    event_channel_name: String,
+    worker: Option<NodeWorkerInstance>,
```

Comment:
could we actually rename it to `node`? worker in worker is a bit confusing

Reply:
sure, renamed :+1: